sdcm/tester.py

```diff
@@ -141,6 +141,43 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class silence:  # pylint: disable=invalid-name
+    test = None
```

Comment:
add docstring what this decorator does and how it can be used

Reply:
added